source/appModules/soffice.py

```diff
@@ -208,11 +209,24 @@ def announceSelectionChange(self):
 
 	def _get_cellCoordsText(self):
 		if self.hasSelection and controlTypes.State.FOCUSED in self.states:
-			selected, count = self.table.IAccessibleTable2Object.selectedCells
-			firstAccessible = selected[0].QueryInterface(IA2.IAccessible2)
+			count = self.table.IAccessibleTable2Object.nSelectedCells
+			selection = self.table.IAccessibleObject.accSelection
+			enumObj = selection.QueryInterface(oleacc.IEnumVARIANT)
+			firstChild, _retrievedCount = enumObj.Next(1)

```

Comment:
```suggestion
			(firstChild, _retrievedCount): Tuple[Union[int, IDispatch], int] = enumObj.Next(1)
```

Reply:
My mistake - it appears you need to add types before unpacking a tuple according to [PEP](https://peps.python.org/pep-0526/#global-and-local-variable-annotations).

Example
```suggestion
			firstChild: Union[int, IDispatch]
			firstChild, _retrievedCount = enumObj.Next(1)
```